UI/CMakeLists.txt

```diff
@@ -57,48 +52,50 @@ configure_file(ui-config.h.in ui-config.h)
 target_sources(
   obs-studio
   PRIVATE # cmake-format: sortable
```

Comment:
Does gersemi parse these? I've seen them removed in other files, but not all files.

Reply:
You mean the `cmake-format` declaration? I think it just ignores them.

Reply:
Yes, the `# cmake-format` declaration. I mostly mean that it seems to have been removed in most other files, but a few were not removed, so I wanted to know if it was intentional.